docs/user/admin.rst

```diff
@@ -147,6 +149,124 @@ Delete User’s Account
   :alt: confirm delete account
 
 
+Manage Classes
+--------------
+
+.. note::
+  To manage Kolibri classes and groups you must be logged-in as **Device Owner** or **Admin**.
+
+You can view, create and delete classes and learner groups, as well as search, filter and enroll Kolibri users in them, using the **Classes** tab in your **Manage** dashboard. Default view displays the list of all classes in your facility, with the number of enrolled users and assigned coaches for each class. 
+
+.. image:: img/classes.png
+  :alt: manage classes
+
+
+Add New Class
+~~~~~~~~~~~~~
+
+To add a new class, follow these steps.
+
+#. Click **Add New Class** button.
+#. Fill in the class name. 
+#. Click **Create** to add the new class or **Cancel** to exit.
+#. Click **Add Users to Class** to :ref:`select_users_class` or **Not now** to simply create the new class. 
+
+
+Delete Class
+~~~~~~~~~~~~
+
+#. Click **Delete Class** button for the chosen class.
+#. Click **Delete** to confirm or **Cancel** to exit without deleting the class. 
+
+.. image:: img/delete_class.png
+  :alt: delete class
+
+.. note::
+  Users enrolled in the class you are deleting will not be removed from the database.
+
+
+Edit Class
+~~~~~~~~~~
+
+To edit a class select it from the default view in the **Classes** tab. In this vew you can organize users in groups, assign roles, add and remove the users from the class. 
+
+
+Add Learner Group
+~~~~~~~~~~~~~~~~~
+
+.. image:: img/learner_groups.png
+  :alt: learner groups list
+
+To add a new learner group, follow these steps.
+
+#. Click **+ New Learner Group** button.
+#. Fill in the group name in the text field. 
+#. Click **Create New Learner Group** button or **Cancel** to exit.
+
+.. image:: img/create_learner_group.png
+  :alt: create learner group
+
+
+Delete Learner Group
+~~~~~~~~~~~~~~~~~~~~
+
+#. Click **Delete Group** button for the chosen group.
+#. Click **Yes** to confirm or **No** to exit without deleting the group.
+
+.. image:: img/delete_learner_group.png
+  :alt: delete learner group
+
+.. note::
+  Users enrolled in the group you are deleting will be assigned the status *Ungrouped*, but will not be removed from the database.
```

Comment:
Oh, nevermind - I see the UI has group as a mutually exclusive drop-down, so it is true. I think under the hood we support multiple group assignment, but I think it makes sense the way you've done at the UI layer.